src/main/java/redis/clients/jedis/params/GeoRadiusStoreParam.java

```diff
@@ -0,0 +1,112 @@
+package redis.clients.jedis.params;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+public class GeoRadiusStoreParam extends Params {
+    private static final String STORE = "store";
```

Comment:
@yangbodong22011 I would have set these (STORE &STOREDIST) as Keywords so you won't have to re-serialize them on each call

Reply:
I'm afraid I don't understand what you mean. Can you give an example?

Reply:
See redis.clients.jedis.Protocol.Keyword